src/main/java/org/jboss/logmanager/handlers/ConsoleHandler.java

```diff
@@ -26,14 +26,13 @@
 
 import java.util.logging.Formatter;
 
+import org.jboss.logmanager.StandardOutputStreams;
 import org.jboss.logmanager.formatters.Formatters;
 
 /**
  * A console handler which writes to {@code System.out} by default.
  */
 public class ConsoleHandler extends OutputStreamHandler {
-    private static final OutputStream out = System.out;
-    private static final OutputStream err = System.err;
```

Comment:
There's no point in storing more objects. Keeping the size smaller is preferred.